contracts/pol/CarbonPOL.sol

```diff
@@ -410,7 +412,9 @@ contract CarbonPOL is ICarbonPOL, Upgradeable, ReentrancyGuardUpgradeable, Utils
 
         // check if the new sale amount is below the current available eth sale amount
         if (newEthSaleAmount < _ethSaleAmount.current) {
-            _ethSaleAmount.current = newEthSaleAmount;
+            _ethSaleAmount.current = address(this).balance > newEthSaleAmount
```

Comment:
Super nitpick: you can use Math.min in all these references

Reply:
Replaced